[2023-05-04T10:43:38.298+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2021-10-07T00:00:00+00:00 [queued]>
[2023-05-04T10:43:38.371+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2021-10-07T00:00:00+00:00 [queued]>
[2023-05-04T10:43:38.373+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 6
[2023-05-04T10:43:38.475+0000] {taskinstance.py:1350} INFO - Executing <Task(BashOperator): third_task> on 2021-10-07 00:00:00+00:00
[2023-05-04T10:43:38.495+0000] {standard_task_runner.py:57} INFO - Started process 3134 to run task
[2023-05-04T10:43:38.536+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'scheduled__2021-10-07T00:00:00+00:00', '--job-id', '837', '--raw', '--subdir', 'DAGS_FOLDER/our_fist_dag.py', '--cfg-path', '/tmp/tmpycalk_am']
[2023-05-04T10:43:38.565+0000] {standard_task_runner.py:85} INFO - Job 837: Subtask third_task
[2023-05-04T10:43:38.736+0000] {task_command.py:410} INFO - Running <TaskInstance: our_first_dag_v3.third_task scheduled__2021-10-07T00:00:00+00:00 [running]> on host a2c266875370
[2023-05-04T10:43:39.109+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='jose-holgado' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2021-10-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-07T00:00:00+00:00'
[2023-05-04T10:43:39.111+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-05-04T10:43:39.112+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task3 and will be running after task1 and with same time as task 2']
[2023-05-04T10:43:39.178+0000] {subprocess.py:86} INFO - Output:
[2023-05-04T10:43:39.189+0000] {subprocess.py:93} INFO - hey, I am task3 and will be running after task1 and with same time as task 2
[2023-05-04T10:43:39.204+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-05-04T10:43:39.381+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, execution_date=20211007T000000, start_date=20230504T104338, end_date=20230504T104339
[2023-05-04T10:43:39.720+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-04T10:43:39.801+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
